packages/payment-integration-api/src/common/utility/amount-transformer.ts

```diff
@@ -0,0 +1,7 @@
+export default class AmountTransformer {
```

Comment:
Will be good to move the `AmountTransformer` migration to a separate PR.
In this new PR, you can also replace all core usage of this functionality with new ones from the package.